packages/modal/src/Modal.tsx

```diff
@@ -88,10 +92,21 @@ const modeStyles: Record<Mode, string> = {
   [Mode.Light]: css`
     color: ${uiColors.gray.dark3};
     background-color: ${uiColors.white};
+    // TODO: Refresh – remove once darkmode is updated
+    font-family: Euclid Circular A, ‘Helvetica Neue’, Helvetica, Arial,
+      sans-serif;
```

Comment:
I think we should merge Group 1 into this so we can use `fontFamilies.default` and `.legacy`